py-polars/polars/series/string.py

```diff
@@ -1636,6 +1636,72 @@ def slice(
         ]
         """
 
+    def head(self, n: int | IntoExprColumn = 10) -> Series:
+        """
+        Return the first n characters of each string in a Utf8 Series.
+
+        Parameters
+        ----------
+        n
+            Length of the slice
+
+        Returns
+        -------
+        Series
+            Series of data type :class:`Struct` with fields of data type :class:`Utf8`.
```

Comment:
This just returns a String type, right?

Reply:
Yes. I am reworking the docstrings and will have an improved version shortly.